CHANGELOG.md

```diff
@@ -29,6 +29,8 @@
 
 - Graphite Scaler: use the latest datapoint returned, not the earliest (https://github.com/kedacore/keda/pull/2365)
 
```

Comment:
Could you delete this line in order to let all the section together? (like in the rest of the file :))

Reply:
Good point. I took the liberty of changing the link format of the previous bullet to that of the template.